tap_google_ads/streams.py

```diff
@@ -441,7 +441,7 @@ def build_stream_metadata(self):
             # Add inclusion metadata
             if self.behavior[report_field]:
                 inclusion = "available"
-                if report_field == "segments.date":
+                if report_field in {"segments.date", "geographic_view.location_type"}:
```

Comment:
We could add a brief comment above here saying something like "Google currently always segments geographic views by the location_type implicitly, so we include it automatically" to help people understand this case since it's a less obvious automatic inclusion.